### DISCUSSION TITLE
navigationGroup and relation manager with dropdown UI

### DISCUSSION THREAD

**Discussion by jechazelle** [filamentphp/filament#11187] [Ideas]
Hi,

I would like to know if it's possible to group the relation from relation manager with dropdown please ?
To add $navigationGroup on RelationManager. Example on this picture, "Relation 7" will be a dropdown with other relations: "Relation 8", "Relation 9"...

<img width="1150" alt="group" src="https://github.com/filamentphp/filament/assets/6713790/be6151e5-3509-4f09-b748-bf499ccb8f00">

Thank you so much for your answer!

**Comment by otavio-araujo** [comment]
I need the same thing.

Did you find the solution?

**Reply by dissto** [reply]
> I need the same thing.
> 
> Did you find the solution?

Not exactly what you are looking for, but you could take a look at [grouping relation managers](https://filamentphp.com/docs/3.x/panels/resources/relation-managers#grouping-relation-managers) (combining multiple RM's into one tab 🤔

**Reply by otavio-araujo** [reply]
Thank you for your reply. But, I want to group them into one Drop-down menu to open individually each one. Grouping them will open all at once in one tab. The problem is that I have too many relation managers and I want to group them to clear space on the tab layout. That's to many tabs on  top of the edit page...

**Comment by otavio-araujo** [comment]
I solved this with Tabs.

![Sem título](https://github.com/user-attachments/assets/8a92bea5-909d-42a4-bf56-bc6421e611ed)

![image](https://github.com/user-attachments/assets/1bbf205c-7186-4e7f-ba43-470f6f8bae6f)

![image](https://github.com/user-attachments/assets/d9ed0839-6e7a-4c28-bf62-9f7f54fc5093)

![image](https://github.com/user-attachments/assets/2f5bed09-4d00-4d28-abcf-5c8ff34f3607)
